tests/008-test-default-lv-data-size.sh

```diff
@@ -0,0 +1,84 @@
+source $SRCDIR/libtest.sh
+
+# Default size of LV that is created from the volume group 
+# by docker-storage-setup is 40% of the free space of the
+# VG, i.e DATA_SIZE=40%FREE. If dss gets modified and changes
+# this default behavior, below code should be modified to 
+# reflect changes
+test_default_lv_size() {
```

Comment:
I think instead of checking default size, it probably will be better if we test for a specific size to make sure DATA_SIZE is working. If it works, then it almost implies that default is working.

So how about testing for say DATA_SIZE=60%FREE in this test case. That way one will also not have to change test case if defaults are updated.